### ISSUE TITLE
New crash on android: java.lang.NoClassDefFoundError - com.google.common.base.CharMatcher

### ISSUE THREAD

**Issue by shaymolho** [airsdk/Adobe-Runtime-Support#2780]
Recently I upgraded some distriqt ANEs and we now see a new crash on android:
[air.com.wizits.buddies_issue_d92aa4f9f4583a271952658f7fe66ca9_crash_session_64E4AAB10278000161B52B00B0AD5A25_DNE_0_v2_stacktrace.txt](https://github.com/airsdk/Adobe-Runtime-Support/files/12408850/air.com.wizits.buddies_issue_d92aa4f9f4583a271952658f7fe66ca9_crash_session_64E4AAB10278000161B52B00B0AD5A25_DNE_0_v2_stacktrace.txt)
This crash affects users with 
Android 6.0.1 
Android 6.0
Android 5.1.1

attached is project.apm and app-descriptor
[project-apm.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/12408853/project-apm.zip)
[slotagram-mobile-boot-app-android.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/12408855/slotagram-mobile-boot-app-android.zip)

**Comment by sjabberwocky** [comment]
I have this on Android 4.4 and 5.1 as well.
It works fine on Android 8.
I do not have Android 6 and 7, but it looks like it only affects Android 6.

Maybe it is related to this issue https://github.com/distriqt/ANE-Firebase/issues/452. (Firebase bug on iOS 12.5).

**Comment by shaymolho** [comment]
I just increased the minimum Android API level to 24 (android 7)

**Comment by marchbold** [comment]
Yeah we are looking into this one, seems older versions of android need a different dependency. Just trying to identify how to best manage this one.

**Comment by marchbold** [comment]
FYI I believe it is the same as this issue : https://github.com/distriqt/ANE-Firebase/issues/455

**Comment by marchbold** [comment]
@sjabberwocky Yeah not a bad option, we recently updated an app to minimum 24 (android 7). 24 is widely considered the minimum for most Android native developers and a lot have moved to 26.  

Currently coverage of these versions stands at:
| API level | Version | Coverage |
| --- | --- | --- |
| 26 | 8 | 93.7%  |
| 24 |  7 | 96.19% | 
| 19 | 4 | 99.64% | 

So you are only gaining a small percentage supporting Android 4, 5, 6

[reference](https://apilevels.com/)

I will look into it either way though.
